SmartDeviceLinkTests/DevAPISpecs/SDLVoiceCommandUpdateOperationSpec.m

```diff
@@ -244,6 +250,46 @@ @interface SDLVoiceCommand()
             });
         });
     });
+
+    // should test voiceCommandsInArray with pendingVoiceCommands being the same as oldVoiceCommands but fewer so that it should only delete
+    it(@"should test voiceCommandsInArray with pendingVoiceCommands being the same as oldVoiceCommands but fewer so that it should only delete", ^{
```

Comment:
It's not good enough to test an internal method. You have to test the _entire_ operation from start to finish that it properly uses the internal method. Please refactor these tests to test the operation, not the internal method.